docs/source/user-guide/sql/operators.md

```diff
@@ -613,3 +613,98 @@ bar") |
 bar         |
 +-----------------+
 ```
+
+## Pipe operators
```

Comment:
Hmm this documentation might belong under `select.md` instead 🤔

Reply:
Fixed in [19d450d](https://github.com/apache/datafusion/pull/17278/commits/19d450d466903eff485760bbd30d55c490dc3dcf)